src/utils/RegHubUtil.js

```diff
@@ -99,7 +112,8 @@ module.exports = {
 
   tags: function (repo, callback) {
     hubUtil.request({
-      url: `${REGHUB2_ENDPOINT}/repositories/${repo}/tags`
+      url: `${REGHUB2_ENDPOINT}/repositories/${repo}/tags`,
+      qs: {'page': 1, 'page_size': 100}
```

Comment:
Don't need to put object keys in quotes - only if they contain `-`.

i.e. you can do `{page: 1, page_size: 100}`